budgeteer-web-interface/src/main/java/org/wickedsource/budgeteer/persistence/contract/ContractEntity.java

```diff
@@ -15,55 +15,59 @@
 import java.util.Set;
 
 @Entity
-@Table(name="CONTRACT")
+@Table(name = "CONTRACT")
 @Data
 public class ContractEntity implements Serializable {
 
-    public enum ContractType{
+    public enum ContractType {
         T_UND_M,
         FIXED_PRICE
     }
 
     @Id
-    @SequenceGenerator(name="SEQ_CONTRACT_ID", sequenceName="SEQ_CONTRACT_ID")
+    @SequenceGenerator(name = "SEQ_CONTRACT_ID", sequenceName = "SEQ_CONTRACT_ID")
     @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "SEQ_CONTRACT_ID")
     private long id;
 
-    @Column(name="CONTRACT_NAME", nullable = false, length = 255)
+    @Column(name = "CONTRACT_NAME", nullable = false, length = 255)
     private String name;
 
     @ManyToOne(optional = false)
     @JoinColumn(name = "PROJECT_ID")
     private ProjectEntity project;
 
     @OneToMany(orphanRemoval = true, cascade = CascadeType.ALL)
-    @JoinColumn(name="CONTRACT_ID")
+    @JoinColumn(name = "CONTRACT_ID")
     private List<ContractFieldEntity> contractFields = new LinkedList<>();
 
-    @Column(name="BUDGET")
+    @Column(name = "SORTING_INDEX")
+    private Integer sortingIndex;
```

Comment:
The column sortingIndex does not fit into the ContractEntity because it's not directly related to a contract.